style.css

```diff
@@ -0,0 +1,112 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500&display=swap');
+:root{
+    --primary-color:#1A2236;
+}
+*,
+*::after,
+*::before{
+    box-sizing: border-box;
+}
+body{
+    margin:0;
+    padding:0;
+    
+    display:flex;
+    flex-direction:column;
+}
+.navbar{
+    width:100%;
+    height:12vh;
+}
+.nav-btn{
+    border:none;
+    background:transparent;
+
+    position:absolute;
+    right: 20px;
+    top: 19px;
+}
+.nav-btn .lines{
+    width: 20px;
+    height: 2px;
+    background: var(--primary-color);
+}
+.nav-btn .line2{
+    margin:3px 0;
+}
+.headline{
+    width:100%;
+    height:88vh;
+
+    display:flex;
+    flex-direction: column;
+}
+.content{
+    width:90%;
+    height:60vh;
+
+    display:flex;
+    flex-direction: column;
+    justify-content: center;
+    margin:0 auto;
+}
+.author{
+    color:var(--primary-color);
+    font-family: 'Playfair Display';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 50px;
+    line-height: 100%;
+
+    margin:0;
+}
+.intro{
+    width:100%;
+    height: 126px;
+
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 21px;
+    letter-spacing: -0.01em;
+
+    margin-top:30px; 
+}
+.collab-btn{
+    background-color:var(--primary-color);
+    color:#fff;
+    font-family: 'Poppins';
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+}
+.collab-btn:hover,.collab-btn:focus{
+    background-color: #fff;
+    color:var(--primary-color);
+    border: 1px solid var(--primary-color);
+}
+.collab-btn:active{
+    border: 2px solid var(--primary-color);
+}
+@media(max-width:500px){
+    .navbar{
+        background-image:url("./img/image_geometry_1.svg");
+        background-repeat:no-repeat;
+    }
+    .author{
+        width:40%;
+        height:100px;
+    }
+    .collab-btn{
+        width:100%;
+        height: 8vh;
+    }
+    .headline{
+        background-image:url("./img/image_geometry_2.svg");
+        background-repeat:no-repeat;
+        background-size:100%;
+        background-position:bottom;
+    }
+}
```

Comment:
- [ ] I can see that you made some changes required in the previous review but you still need to work on something, but you are almost there, kindly remove that space between your content and your background-image so that your website will look professional :wink: 
Your design 
![space2](https://user-images.githubusercontent.com/88894525/204974346-edeca4e9-11ea-459d-b1c7-75ac16b6579f.png)


Figma Design: 
![space1](https://user-images.githubusercontent.com/88894525/204974290-e2cffa51-f85c-4e4c-98a8-911ad7758380.png)